# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../koffice/koffice-1.5.1-kexi-sqlite2-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur koffice-1.5.1-orig/kexi/kexidb/drivers/sqlite2/Makefile.am koffice-1.5.1/kexi/kexidb/drivers/sqlite2/Makefile.am
--- koffice-1.5.1-orig/kexi/kexidb/drivers/sqlite2/Makefile.am	2006-05-10 18:21:29.000000000 +0000
+++ koffice-1.5.1/kexi/kexidb/drivers/sqlite2/Makefile.am	2006-06-10 17:14:42.000000000 +0000
@@ -23,10 +23,8 @@
 kexidb_sqlite2driver_la_LDFLAGS = $(all_libraries) -module $(KDE_PLUGIN) $(VER_INFO) 
 
 # we need the .mocs to be built
-sqliteconnection.lo: ../sqlite/sqliteconnection.lo
-sqlitecursor.lo: ../sqlite/sqlitecursor.lo
-sqlitedriver.lo: ../sqlite/sqlitedriver.lo
-sqlitepreparedstatement.lo: ../sqlite/sqlitepreparedstatement.lo
+sqliteconnection.lo: ../sqlite/sqliteconnection.moc
+sqlitedriver.lo: ../sqlite/sqlitedriver.moc
 
 kde_services_DATA = kexidb_sqlite2driver.desktop
 
